ui/components/component-library/avatar-account/avatar-account.tsx

```diff
@@ -14,6 +18,21 @@ import {
   AvatarAccountProps,
 } from './avatar-account.types';
 
+// TODO: This might not scale well with our new multichain initiative since it would require
```

Comment:
you can make it a little more ready for extensibility with an approach like that. wdyt?
```
type ChainType = 'btc' | 'eth';

function getChainType(address: string): ChainType | undefined {
  if (isBtcMainnetAddress(address) || isBtcTestnetAddress(address)) {
    return 'btc';
  }
  // Default to 'eth' for other cases, can be extended as needed
  return 'eth';
}

function getJazziconNamespace(address: string): string | undefined {
  const chainType = getChainType(address);

  switch (chainType) {
    case 'btc':
      return 'bip122';
    case 'eth':
      return undefined; // Falls back to default Jazzicon behavior
    default:
      return undefined;
  }
}

Reply:
I like this idea, lemme address this now!